tests/unit/transports/test_sms.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+from flask import Response
+
+from funnel.transports.base import TransportTransactionError
+from funnel.transports.sms import SmsSender
+
+
+class TestSmsSender:
+    """ Tests for SMS Sender """
+
+    # Target Numbers (Test Only). See this
+    # https://www.twilio.com/docs/iam/test-credentials
+    TWILIO_CLEAN_TARGET = "+15005550010"
+    TWILIO_INVALID_TARGET = "+15005550001"
+    TWILIO_CANT_ROUTE = "+15005550002"
+    TWILIO_NO_SMS_SERVICE = "+15005550009"
+
+    # Dummy Message
+    MESSAGE = "Test Message"
+
+    def test_twilio_success(self):
+        """ Test if Message sending is a success. """
+        sender = SmsSender(callback=False)
```

Comment:
This `callback` parameter is not used in the singleton. Also, since it's a singleton, you must test with the existing singleton for tests to match runtime expectations